modules/balana-core/src/main/java/org/wso2/balana/xacml3/AnyOfSelection.java

```diff
@@ -131,6 +131,10 @@ public MatchResult match(EvaluationCtx context) {
         }
     }
 
+    public List<AllOfSelection> getAllOfSelections() {
+        return allOfSelections;
```

Comment:
Best practice would be to return the unmodifiable list using 'Collections.unmodifiableList'